.circleci/config.yml

```diff
@@ -181,10 +178,7 @@ jobs:
         name: Docker Build
         # Run MYSQL image and build v2xhub docker image develop image
         command: |
-          cd configuration/amd64/
-          docker-compose up -d db
-          cd ../../
-          docker build --network=host -t usdotfhwaops/v2xhubamd:latest .
+          docker build -t usdotfhwaops/v2xhubamd:latest .
```

Comment:
Same comment as above. Please confirm docker-compose no longer need to be run.

Reply:
It is no longer needed. It was needed before since we were installing plugins/ adding configurations to mysql during the build process. We know only do that on startup. This allows the build image not to need or be bound to mysql credentials.